### PR TITLE
SC-7878 - dropdowns and multiselects

### PR DESCRIPTION
**Pull request by haniapalac**
# Description

## Links to Tickets or other pull requests

https://ticketsystem.hpi-schul-cloud.org/browse/SC-7878

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: views/courses/course.hbs [hunk 1]
```diff
@@ -57,18 +57,18 @@
...
       {{$t "courses._course.link.toCourseFiles"}}
     </a>
   </div>
 </div>
 <section class="section-course">
   <div class="tabContainer" data-active-tab={{activeTab}}>
     <div class="tabs" data-max-width="1024">
-      <button class="tab" data-testid="topics" data-tab="js-topics">
-        <span class="text">{{$t "courses._course.tab_label.topics"}}
+      <button class="tab" data-testid="topics" data-tab="js-topics" type="button">
```
Comment (CordlessWool) [inline views/courses/course.hbs:68]: type button at a button element? Is this necassery?

File: views/courses/course.hbs [hunk 2]
```diff
@@ -57,18 +57,18 @@
...
+        <span class="text">{{$t "courses._course.tab_label.topics"}}</span>
       </button>
-      <button class="tab" data-testid="hometasks" data-tab="js-homeworks">
+      <button class="tab" data-testid="hometasks" data-tab="js-homeworks" type="button">
```
Comment (CordlessWool) [inline views/courses/course.hbs:71]: same like above

File: views/courses/course.hbs (hunk 1 above)
Comment (CordlessWool) [reply views/courses/course.hbs:68]: see hole file (other buttons do not have this)

**Review by mamutmk5 — DISMISSED** [review]

File: views/courses/create-course.hbs [hunk 3]
```diff
@@ -190,22 +197,23 @@
...
                              {{#each ../students}}
-                              <option value="{{this._id}}" {{#if this.selected}}selected{{/if}}>
+                                <option value="{{this._id}}" {{#if this.selected}}selected{{/if}}>
                                   {{#if this.displayName}}
                                       {{this.displayName}}
                                   {{else}}
                                       {{this.firstName}} {{this.lastName}}
                                   {{/if}}
-                              </option>
+                                </option>
```
Comment (CordlessWool) [inline views/courses/create-course.hbs:212]: linter: to much tabs

File: views/courses/edit-course.hbs [hunk 4]
```diff
@@ -128,9 +119,9 @@
 				</div>
 
 				<div class="form-group">
-					<label>{{$t "courses.global.label.courseTeacher"}}</label>
+					<label for="teachers">{{$t "courses.global.label.courseTeacher"}}</label>
 
-					<select name="teacherIds[]" required multiple data-placeholder="{{$t "courses.global.input.selectStudents"}}"
+					<select id="teachers" name="teacherIds[]" required multiple data-placeholder="{{$t "courses.global.input.selectStudents"}}"
```
Comment (CordlessWool) [inline views/courses/edit-course.hbs:124]: more spezic id

Comment (CordlessWool) [reply views/courses/edit-course.hbs:124]: some for other ids

**Review by CordlessWool — CHANGES_REQUESTED** [review]

File: locales/en.json [hunk 5]
```diff
@@ -890,7 +891,8 @@
 				"text": {
 					"attentionDeletion": "All data will be permanently deleted.",
 					"courseWillDelete": "This also deletes the course for all students.",
-					"toEditChangeDate": "In order to edit course data, you must first adjust the end date."
+					"toEditChangeDate": "In order to edit course data, you must first adjust the end date.",
+					"hintSelection": "Um deine Auswahl zu treffen, halte bitte die Tasten cmd + shift gedrückt.\nDeine Auswahl erscheint dann grau hinterlegt. Solltest du weitere Hilfe benötigen wende dich bitte an den User Support."
```
[the remark this answers is not in the document]
Comment (haniapalac) [reply locales/en.json:895]: @luizzaly could you provide the translation?

**Pull request merged**